com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1371,48 +1377,26 @@ internal void HandleApproval(ulong ownerClientId, bool createPlayerObject, uint?
                     ConnectedClients[ownerClientId].PlayerObject = networkObject;
                 }
 
-                m_ObservedObjects.Clear();
-
-                foreach (var sobj in SpawnManager.SpawnedObjectsList)
-                {
-                    if (ownerClientId == ServerClientId || sobj.CheckObjectVisibility == null || sobj.CheckObjectVisibility(ownerClientId))
-                    {
-                        m_ObservedObjects.Add(sobj);
-                        sobj.Observers.Add(ownerClientId);
-                    }
-                }
-
+                // Don't send the CONNECTION_APPROVED message if this is the host that connected locally
                 if (ownerClientId != ServerClientId)
                 {
-                    // Don't send any data over the wire if the host "connected"
-                    ulong[] clientIds = { ownerClientId };
-
-                    var context = MessageQueueContainer.EnterInternalCommandContext(
-                        MessageQueueContainer.MessageType.ConnectionApproved, NetworkChannel.Internal,
-                        clientIds, NetworkUpdateStage.EarlyUpdate);
+                    var context = MessageQueueContainer.EnterInternalCommandContext(MessageQueueContainer.MessageType.ConnectionApproved, NetworkChannel.Internal, new ulong[] { ownerClientId },
+                        NetworkUpdateStage.EarlyUpdate);
 
                     if (context != null)
                     {
                         using (var nonNullContext = (InternalCommandContext)context)
                         {
                             nonNullContext.NetworkWriter.WriteUInt64Packed(ownerClientId);
-
-                            if (NetworkConfig.EnableSceneManagement)
-                            {
-                                nonNullContext.NetworkWriter.WriteUInt32Packed(NetworkSceneManager.CurrentSceneIndex);
-                                nonNullContext.NetworkWriter.WriteByteArray(NetworkSceneManager.CurrentSceneSwitchProgressGuid
-                                    .ToByteArray());
-                            }
-
                             nonNullContext.NetworkWriter.WriteInt32Packed(LocalTime.Tick);
-                            nonNullContext.NetworkWriter.WriteUInt32Packed((uint)m_ObservedObjects.Count);
-
-                            for (int i = 0; i < m_ObservedObjects.Count; i++)
-                            {
-                                m_ObservedObjects[i].SerializeSceneObject(nonNullContext.NetworkWriter, ownerClientId);
-                            }
                         }
                     }
+
+                    // Now inform the newly joined client of the scenes to be loaded as well as synchronize it with all relevant in-scene and dynamically spawned NetworkObjects
+                    if (NetworkConfig.EnableSceneManagement)
+                    {
+                        SceneManager.SynchronizeNetworkObjects(ownerClientId);
```

Comment:
Just for my own understanding before regardless of EnableSceneManagement being set we synced objects now we only sync them if its set... is this behavioral change?

Reply:
just to add onto that, are we basically implying that if SceneManagement is disabled, we will not sync in-scene-placed NetworkObjects?

Reply:
So I am concerned about this change, because to me its a significant change in behavior. If I am a developer and I am building a level and lets say I place powerups in my level or things that need to be synced. Before it was expected those those would just work regardless of NGO was managing the scene transitions or not. Now we are saying well that is not the case, we are breaking the contract of what a NetworkObject means when you place it on something in your scene. We are essentially saying now well its only a NetworkObject in the way it was before if you let NGO manage your scenes and if you don’t then well its not really a NetworkObject in the same way as it was before. To me this is a breaking change and I think changes what a dev may have as an expectation of NetworkObjects in their scene. 

I am curious @mattwalsh-unity what your thoughts are on this because to be this could just break peoples games that are currently not relying on scene management being done by NGO.

Reply:
I think another huddle or zoom meeting would be great.  Then I can walk you both through the NetworkObject synchronization process.  There are actually many breaking changes in v1.0 that will force users to have to re-think/refactor any projects currently underway, but the good news is that if they are handling their own scene management then they are more than likely mid-level to advanced users and they will have full access to this source code and our support to help them!  :)

Reply:
Are we saying that those other changes actually just totally break peoples games from working the way they did before?

Reply:
Also, I will get with Jeffrey to see how far we are from the Snapshot system taking over this.
If it looks like it might be after v1.0, then I will just add an optional flag at the end of the HandleApproval serialization stream that denotes whether there are NetworkObjects to be synchronized or not (i.e. scene management disabled) and if so it will perform the exact same synchronization as the v0.1.1 did.

Reply:
@andrews-unity 
If you check the most recent update, when scene management is disabled it will now properly synchronize NetworkObjects the same way it has done it in the past during the approval process.  Let me know if you have any other concerns regarding this?  (Good catch BTW)